src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/services/DmnTable.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.services;
+
+public enum DmnTable {
+
+    TASK_CANCELLATION("wa-task-cancellation"),
+    TASK_INITIATION("wa-task-initiation");
+
+    private final String tableName;
+
+    DmnTable(String tableName) {
+        this.tableName = tableName;
+    }
+
+    public String getTableName() {
+        return tableName;
+    }
+
+    public String getTableKey(String jurisdictionId, String caseTypeId) {
+        return tableName + "-" + jurisdictionId + "-" + caseTypeId;
```

Comment:
one small suggestion here as it has happened to me before, 
Shall we add some Object.requireNonNull for jurisdiction and casetypeId ?

Reply:
There is validation in the EventInformation when this message is received from CCD. Jurisdiction and case type cannot be empty. Thanks.